### DISCUSSION TITLE
zh/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress/marketplace#10] [Announcements]
# zh/

升级到最新的 VuePress2 添加你的主题和插件 请参阅 在此表达你的反馈 

https://marketplace.vuejs.press/zh/

**Comment by zlzlezhi** [comment]
**有人吗**

**Reply by Mister-Hope** [reply]
请尽量避免发送无意义的信息，这不是一个测试用例。

**Comment by wakakaawsl** [comment]
vuepress2.x版本能嵌入视频吗，没有找到相关文档或操作方法

**Comment by lfx1993** [comment]
对于下面这种，应该用哪个插件可以解析，实现示例代码块的效果显示及折叠代码块功能
::: demo 这里是代码块的标题
```vue
<template>
  <div>{{ message }}</div>
</template>
 
<script>
export default {
  data() {
    return {
      message: 'Hello VuePress!',
    };
  },
};
</script>

**Comment by zb-zd** [comment]
vuepress2项目，直接打开除首页外的其他layout页面都会报错，Hydration completed but contains mismatches.
复现步骤：
1. npm init vuepress vuepress-starter
2. 打包部署上线
3. 导航栏点击category
4. 刷新网站，控制台报错Hydration completed but contains mismatches.
如果使用自定义首页，问题会更严重，页面渲染异常

**Comment by Vogadero** [comment]
请问导航栏如何改成使用icon或者图标来显示，而不是单纯文字？

**Comment by Vogadero** [comment]
有个bug，这个页面的主题按钮点击跳转的是https://marketplace.vuejs.press/zh/Themes/会报404，导航栏的主题跳转的是https://marketplace.vuejs.press/zh/themes/这才是正常的
